client/gui_rpc_server_ops.cpp

```diff
@@ -775,7 +775,8 @@ void handle_get_project_config_poll(GUI_RPC_CONN& grc) {
         );
     } else {
         const char *p = grc.get_project_config_op.reply.c_str();
-        const char *q = strstr(p, "<project_config");
+        const char *q = strstr(p, "<project_config>");
+        if (!q) q = strstr(p, "<error>");
         if (!q) q = "<project_config/>\n";
```

Comment:
This line is now dead code and is never reached. The output now also changes from `<project_config/>` to `<error>` which breaks code that expects the old behavior.